handlers/providers/rackspace/rackspace.go

```diff
@@ -0,0 +1,29 @@
+package rackspace
+
+import (
+	log "github.com/Sirupsen/logrus"
+	"github.com/rancherio/go-machine-service/handlers/providers"
+	"github.com/rancherio/go-rancher/client"
+)
+
+func init() {
+	rackspaceHandler := &RackspaceHandler{}
+	if err := providers.RegisterProvider("rackspace", rackspaceHandler); err != nil {
+		log.Fatal("could not register rackspace provider")
+	}
+}
+
+type RackspaceHandler struct {
+}
+
+func (*RackspaceHandler) HandleCreate(machine *client.Machine, machineDir string) error {
+	return nil
+}
+
+func (*RackspaceHandler) HandleError(msg string) string {
+	prettyMsg := msg
+	if msg == "Expected HTTP response code [200 203] when accessing [POST https://identity.api.rackspacecloud.com/v2.0/tokens], but got 401 instead" {
```

Comment:
Ran some tests and when there is a 401, this is what we get (in other cases as well). If the API changes in a newer version of machine, updating this would be a part of supporting that, but after speaking to @will-chan, i understand that cloud provider's API rarely changes. Can you confirm that once again here Will?